### DISCUSSION TITLE
curled leg R

### DISCUSSION THREAD

**Discussion by Ayuths** [rsms/inter#735] [Ideas]
![IMG_3496](https://github.com/user-attachments/assets/dafcd223-8322-4153-97b1-8171bfb66aea)
Curled leg R like in helvetica

**Comment by DGCK81LNN** [comment]
Hello friend, welcome to GitHub. However please keep in mind that GitHub issues and discussions are for issue reports and constructive feedback. Your message doesn't clearly express anything and is likely to be considered meaningless. In order to keep this place a productive and efficient environment dedicated to development, try not to post random things like this in the future. Instead, clearly express your opinions and kindly ask questions.
